src/Storefront/Framework/Twig/TemplateDataExtension.php

```diff
@@ -61,7 +61,9 @@ public function getGlobals(): array
                 'showStagingBanner' => $this->showStagingBanner,
             ],
             'themeId' => $themeId, /** Not used in Twig template directly, but in @see \Shopware\Storefront\Framework\Twig\Extension\ConfigExtension::getThemeId */
+            /** @deprecated tag:v6.8.0 - Will be removed in 6.8.0 as it will be replaces with the "activeRoute" variable */
```

Comment:
```suggestion
            /** @deprecated tag:v6.8.0 - Will be removed. Use the "activeRoute" variable instead */
```

that should be enough. the tag is already saying, when it is going to be removed